soccer/src/rj_vision_filter/camera/camera.cpp

```diff
@@ -163,7 +163,7 @@ void Camera::update_balls_mhkf(RJ::Time calc_time, const std::vector<CameraBall>
         const CameraBall& camera_ball = ball_list.at(i);
         bool was_used = used_camera_ball.at(i);
 
-        if (!was_used && kalman_ball_list_.size() < PARAM_max_num_kalman_balls) {
+        if (!was_used && (int64_t) kalman_ball_list_.size() < PARAM_max_num_kalman_balls) {
```

Comment:
```suggestion
        if (!was_used &&  static_cast<int64_t>(kalman_ball_list_.size()) < PARAM_max_num_kalman_balls) {
```
or you could maybe do

```suggestion
        if (!was_used &&  static_cast<decltype(PARAM_max_num_kalman_balls)>(kalman_ball_list_.size()) < PARAM_max_num_kalman_balls) {
```
i'm not sure the best practice, anything is fine really, but no C-style casts please.